ckanext/switzerland/helpers.py

```diff
@@ -265,3 +265,31 @@ def ogdch_localised_number(number):
         return numbers.format_number(number, locale='de_CH')
     else:
         return localised_number(number)
+
+
+def ogdch_group_tree(type_='organization'):
+    organizations = tk.get_action('group_tree')(
+        {},
+        {'type': type_, 'all_fields': True}
+    )
+    organizations = get_sorted_orgs_by_translated_title(organizations)
+    return organizations
+
+
+def get_sorted_orgs_by_translated_title(organizations):
+    for organization in organizations:
+        organization['title'] = \
+            set_translated_group_title(organization['title'])
+        if organization['children']:
+            get_sorted_orgs_by_translated_title(organization['children'])
```

Comment:
isn't here a return statement needed? or are the organization passed by reference?